app/views/screens/LoginScreen.tsx

```diff
@@ -78,28 +79,30 @@ function LoginScreen(props) {
         <View style={styles.form}>
           <TextInput
             label="Username"
-            value={username}
-            onChangeText={(text) => {
-              errorMessage && setErrorMessage("");
-              setUsername(text);
-            }}
+            name="username"
+            control={control}
+            rules={{ required: "Username is required" }}
           />
           <TextInput
             label="Password"
-            value={password}
+            name="password"
+            control={control}
             secureTextEntry={true}
             textContentType="password"
-            onChangeText={(text) => {
-              errorMessage && setErrorMessage("");
-              setPassword(text);
+            rules={{
+              required: "Password is required",
+              minLength: {
+                value: 8,
+                message: "Password should be at least 8 characters long",
+              },
```

Comment:
Hey, backend have this criteria, can we do the same here? https://github.com/ndedhia1210/smallBusiness/blob/main/app/user/resetUserPassword.js#L29

Password must be alphanumeric length can be from 5-10 (We can change BE to make it 8-12 length but for now can we use this and check for alpha numeric too)

Reply:
Done! Required, 5-10 chars, alphanumeric.